src/Mod/PartDesign/Gui/TaskExtrudeParameters.h

```diff
@@ -57,13 +63,23 @@ class TaskExtrudeParameters : public TaskSketchBasedParameters
         Pocket
     };
 
+    enum class SidesMode {
+        OneSide,
+        TwoSides,
+        Symmetric,
+    };
+
+    enum class Sides {
+        First,
+        Second,
+    };

```

Comment:
```suggestion
    enum class Side {
        First,
        Second,
    };
```

enums should be singular. "Which Side? First"

Reply:
ok